frontend-react/src/styles/_uswds_settings.scss

```diff
@@ -1,73 +1,63 @@
 // USWDS Settings that are different from the defaults.
 // see: https://designsystem.digital.gov/documentation/settings/
 
-/*
-  These are necessary for the image and font urls referenced in the source
-  files to resolve correctly.
-
-
-    these paths are from where the output css lives, in this case
-            /frontend-react/src/content/generated/global.out.css
- */
-$theme-image-path: "../../../node_modules/@uswds/uswds/dist/img";
-$theme-font-path: "../../../node_modules/@uswds/uswds/dist/fonts";
-$theme-hero-image: "../../../node_modules/@uswds/uswds/dist/img/hero.jpg";
-
-$theme-show-compile-warnings: true;
-$theme-show-notifications: false;
-$theme-font-role-heading: "sans";
-
-// Override defaultUSWDS maximum width settings
-$theme-header-max-width: 'desktop';
-// $theme-header-min-width: 'desktop-lg';
-
-$theme-grid-container-max-width: 'desktop';
-$theme-site-margins-breakpoint: 'desktop';
-$theme-site-alert-max-width: 'desktop';
-// $theme-identifier-max-width: 'desktop-lg';
-
-$theme-footer-max-width: 'desktop';
-
-$theme-site-margins-width: 4;
-$theme-site-margins-mobile-width: 2;
-
-//Typography
-$theme-font-type-cond:   false;
-$theme-font-type-icon:   false;
-$theme-font-type-lang:   false;
-$theme-font-type-mono:   "roboto-mono";
-$theme-font-type-sans:   "public-sans";
-$theme-font-type-serif:   false;
-
-$theme-font-role-ui:      "sans";
-$theme-font-role-heading: "sans";
-$theme-font-role-body:    "sans";
-$theme-font-role-code:    "mono";
-$theme-font-role-alt:     "mono";
-
-// Breakpoints
-$mobile: 480px;
-$tablet: 1024px;
-$desktop: 1200px;
-
-// Flex
-
-$flex-settings: (
-  responsive: true
-);
-
-$flex-direction-settings: (
-  responsive: true
+@use "uswds-core" with (
+  /*
+    These are necessary for the image and font urls referenced in the source
+    files to resolve correctly.
+
+
+      these paths are from where the output css lives, in this case
+              /frontend-react/src/content/generated/global.out.css
+  */
+  $theme-image-path: '@uswds/uswds/img',
```

Comment:
Is this the same image as what was in '@uswds/uswds/dist/img'?

Reply:
if you mean $theme-hero-image, the png was replaced for a jpg at some point. since we're seemingly not changing the image from what uswds supplies, I just removed the variable from our file.